JWTManipulator9000/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
All projects hosted by Layer5 are Apache v2 licensed (see https://layer5.io/company/faq). You can replace the MIT license by including the following in cargo.toml:

```
// Copyright 2021 The Layer5 Authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Reply:
will do :)

Reply:
Is it fine if I update the license directly?